internal/quickstart/messages.go

```diff
@@ -24,6 +24,25 @@ func sendErr(err error) tea.Cmd {
 	}
 }
 
+type toggledFlagMsg struct{}
+
+func sendToggleFlagMsg(client flags.Client, accessToken, baseUri, flagKey string, enabled bool) tea.Cmd {
```

Comment:
Does it make more sense to name this after the message it sends, so in this case it would be `sendToggledFlagMsg` (past tense)? I could see it implicitly sending a `toggleFlagMsg` and returning the results, either a `toggledFlagMsg` or `errMsg`, but I could also see that as confusing because there isn't a specific `toggleFlagMsg` type.

Reply:
ya I did actually get a bit tripped on this looking for the the actual msg type it specified when I picked it back up again, but it is also Doing The Thing here